### PR TITLE
16376 Fix pending limited restoration blurb

### PR DESCRIPTION
**Pull request by leodube-aot**
*Issue #:* /bcgov/entity#16376

*Description of changes:*

- Ensure that the filing object is not pending before showing the item details. If it is pending use the default template.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

**Review by severinbeauvais — APPROVED** [review]
LGTM.

Were you able to test this?

Remember to update the app version and commit those 2 files, too.

**Comment by leodube-aot** [comment]
> LGTM.
> 
> 
> 
> Were you able to test this?
> 
> 
> 
> Remember to update the app version and commit those 2 files, too.

Yeah I was able to test it similarly to the last ticket I worked on.

I'll update the app version and commit the 2 files (I keep forgetting haha)

**Comment by leodube-aot** [comment]
> Looks like this needs to be rebased, please.

I may need a hand with the rebase since I can't seem to get it to work.

**Review by JazzarKarim — APPROVED** [review]
LGTM!

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-filings-ui",
-  "version": "6.5.12",
+  "version": "6.5.14",
```
Comment (JazzarKarim) [inline package.json:3]: Wait, 2 jumps here? I think it should be `6.5.13`.

Comment (severinbeauvais) [reply package.json:3]: Hongjing has a PR updating this to 13.

There's no easy solution here. If Hongjing merges before you then you'll need to rebase and you'll have a merge conflict. If you merge first then we skip a version (not a big deal).

**The main reason for the app version is so we can easily check in Dev/Test/Prod which version is running, so you can tell if your code is there or not.**

**Review by severinbeauvais — APPROVED** [review]
LGTM

**Review by ketaki-deodhar — APPROVED** [review]
Looks good

File: package.json (hunk 1 above)
Comment (JazzarKarim) [reply package.json:3]: Oh OK. I thought this PR is going to be merged first. Sorry about that, my bad! Resolving 👍

Comment (severinbeauvais) [reply package.json:3]: This PR was ready before Hongjing's so I went ahead and merged it.

Leo, after the GitHub CD action has completed, verify this in Dev and then move your ticket to UX Assurance/Verification. I don't know if it's going to be possible to see this in action in Dev, but maybe the UX folks will be satisfied with a screenshot from you?

Comment (severinbeauvais) [reply package.json:3]: PS ticket workflow:
- In Progress: you're working on it
- Review: your PR is ready and bring reviewed
- Staging: your PR has been merged and you have yet to verify in Dev
- UX Assurance/Verification: the UX folks want to look at it (denoted by UX Assurance label on ticket)
- Ready for QA: the UX folks didn't want to look at it, or they looked at it and are happy

**Pull request merged**